src/System.Private.CoreLib/src/System/Threading/Tasks/TPLETWProvider.cs

```diff
@@ -33,6 +33,8 @@ internal sealed class TplEtwProvider : EventSource
         internal bool Debug;
         private bool DebugActivityId;
 
+        private const int appDomainID = 1;
```

Comment:
Nit: Constants should be PascalCase (https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/coding-style.md)

Reply:
`DefaultAppDomainID` ?